core/templates/domain/question/QuestionObjectFactory.ts

```diff
@@ -35,15 +37,15 @@ export interface QuestionBackendDict {
 }
 
 export class Question {
-  _id: string;
+  _id: string | null;
```

Comment:
The `null` should be explained by a comment.

Reply:
Done

Reply:
Looks good, add a similar comment above.

Reply:
is it okay if i add TODO comment before `createDefaultQuestion` (line no. 213) for:   Remove the default question so that question can be created from start.

because `createDefaultQuestion` is used in 4 files and refactor code seems complicated,

Reply:
Sure.

Reply:
Done